### PR TITLE
fix some issues with assets

### PR DESCRIPTION
**Pull request by zakmat**
fixed some broken paths for a texture
removed Imu prefab - it fails without Rigid Body component

### CODE REVIEW

File: Gems/RosRobotSample/Assets/robot/rosbot_xl_description/meshes/base.material [hunk 1]
```diff
@@ -2,23 +2,23 @@
     "materialType": "@gemroot:Atom_Feature_Common@/Assets/Materials/Types/StandardPBR.materialtype",
```
Comment (pawelbudziszewski) [inline Gems/RosRobotSample/Assets/robot/rosbot_xl_description/meshes/base.material:2]: I can't find any place where this material is used. RosRobot uses this one: `RosRobotSample/Assets/robot/Rosbot_Mat.material` (which, BTW, looks identical). If it's not used, we can remove it.

**Comment by michalpelka** [comment]
I removed IMU prefab as well in #303

File: Gems/RosRobotSample/Assets/robot/rosbot_xl_description/meshes/base.material (hunk 1 above)
Comment (adamdbrw) [reply Gems/RosRobotSample/Assets/robot/rosbot_xl_description/meshes/base.material:2]: @zakmat could you remove this one as well?

**Review by adamdbrw — APPROVED** [review]

File: Gems/RosRobotSample/Assets/robot/rosbot_xl_description/meshes/base.material (hunk 1 above)
Comment (zakmat) [reply Gems/RosRobotSample/Assets/robot/rosbot_xl_description/meshes/base.material]: All right, I have removed it

**Comment by adamdbrw** [comment]
@zakmat @michalpelka is this one still needed after recent fixes?

**Comment by michalpelka** [comment]
IHMO it is not needed.

**Comment by zakmat** [comment]
Failing prefab was IMO the biggest issue - and was already fixed in #303 - unused material might be removed as a part of another clean-up session

**Pull request closed without merging**